### PR TITLE
Fixed issues in Readme

### PR DESCRIPTION
**Pull request by OutCraft-Mods**
Hi, I fixed some small issues in the Readme file and rewrote the `How to embed this Mod` section to be more easy to understand to other Modders who might not be too familiar with Gradle

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ This mod helps modders to bring attention to players about the StopModReposts ca
 ---
 # Forge Version
 
-## What does 'StopModReposts' do?
-It adds this beautiful GUI, in order to educate more users about the problem of mod reposting sites.
+## What does this Mod do?
+StopModReposts adds this beautiful GUI, in order to educate more users about the problem of mod reposting sites.
```
Comment (berrysauce) [inline README.md]: Please change this to: „**The StopModReposts Splash-Screen Mod** adds this beautiful GUI (…)“

**Review by berrysauce — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (OutCraft-Mods) [reply README.md]: Done, what about the title `What does this Mod do?`, is that fine?

Comment (berrysauce) [reply README.md]: That‘s fine! I‘m just taking a quick look over the rest and then I can approve it! Thanks so much 🙏

Comment (OutCraft-Mods) [reply README.md]: Always happy to help 😁

**Review by berrysauce — APPROVED** [review]

**Pull request merged**